pos_order_list/models/pos_config.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 GRAP - Sylvain LE GAL
+# Copyright 2018 Tecnativa S.L. - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    iface_load_done_order = fields.Boolean(
+        string='Load Done Orders',
+        default=False,
```

Comment:
```suggestion
        default=True,
```

We can guess that if users install the module, he want to see the feature enabled.